capo-simple.json

```diff
@@ -6,217 +6,24 @@
         "basicPropertyValues": [
           {
             "pred": "http://purl.org/dc/terms/description",
-            "val": "None"
+            "val": "An ontology that contains CAP related ontology terms gathered from different ontologies "
           },
           {
             "pred": "http://purl.org/dc/terms/license",
-            "val": "https://creativecommons.org/licenses/unspecified"
+            "val": "http://creativecommons.org/licenses/by/4.0/"
```

Comment:
Same, why not `https` here?

Reply:
Took the url from an other ontology edit file, I guess it should have been https instead http. changed it to https.